Clarify name resolution for source and target socket

TimWolla opened the pull request.

TimWolla commented on spiped/README:
```diff
@@ -21,9 +21,12 @@ Options:
+	accept at the expired address.
     -t <target socket>
-	Address to which spiped should connect.
+	Address to which spiped should connect. The accepted formats are the
```
I did not adapt the docs for *spipe* yet. I think it might be best to copy lines 17 to 21 of this file into the docs of *spipe*. What do you think?

gperciva replied:
The style for cperciva man-pages is to have two spaces after a period.

gperciva replied:
I agree that the example of address formats should be available in the docs for `spipe`.  I'm not certain why spiped has so many different documentation files:
- `/README`
- `/README.md`
- `/spiped/README`
- `/spiped/spiped.1`
- `/spipe/README`
- `/spipe/spipe.1`

My first instinct is to remove the `/*/README` files since that info seems to be the the man-pages, and to rename the top-level README into something else.  I'll discuss this with @cperciva offline (unless he comments in the next 3 hours).

cperciva replied:
History: I wrote README files, then someone turned them into man pages; then spiped came to github and someone decided that we should have documentation in markdown format...

TimWolla replied:
> My first instinct is to remove the /*/README files since that info seems to be the the man-pages

I agree on this. If the idea of my changes is fine I'll fix the stylistic and grammatical changes, remove the redundant `README` files and update this pull request?

gperciva replied:
That sounds good, thanks!

gperciva commented on spiped/README:
```diff
@@ -21,9 +21,12 @@ Options:
 	Note that hostnames are resolved when spiped is launched and are not
 	re-resolved later; thus if DNS entries change spiped will continue to
-	connect to the expired address.
+	accept at the expired address.
```
Grammatically speaking, I think that "accept connections at" would be more clear.

gperciva commented on spiped/README:
```diff
@@ -21,9 +21,12 @@ Options:
     -t <target socket>
-	Address to which spiped should connect.
+	Address to which spiped should connect. The accepted formats are the
+	same as the ones accepted by <source socket>.
```
`man` will word-wrap this paragraph, so I suggest removing the newline and putting "Contrary" on this same line.

gperciva replied:
My apologies, I just noticed that this is a README, not a man-page.  I should have made the comments about the man-page, although by symmetry the changes should probably also be made in the README, so the basic idea remains the same.

TimWolla replied:
The manpage looks pretty strange already because of that word wrapping. Especially the list of accepted formats. I'll try fix that if the textual changes are fine (see my other comment).

TimWolla commented:
I just pushed a new commit. The man pages were tested using GNU man 2.7.5 on Ubuntu Xenial. I made the following changes:

- Removed the redundant README files
- Unified the man pages of spipe(1) and spiped(1) (in regard to `-t`)
- Improved formatting to be consistent with itself and other man pages
- Improved wording of `-t` and `-s` for spiped(1).

cperciva commented:
The result looks good to me, but this is a lot in a single commit; can you split it up?

TimWolla commented:
> The result looks good to me, but this is a lot in a single commit; can you split it up?

Sure. I fixed a few mistakes I did in the first version as well (and added myself in the copyright notice; BTW: You might want to update the COPYRIGHT file.)

gperciva commented on spipe/spipe.1:
```diff
@@ -23,7 +23,7 @@
 .\" SUCH DAMAGE.
```
The commit message contains "theother"; should have a space in it.

gperciva commented on spiped/spiped.1:
```diff
@@ -43,12 +43,17 @@ spiped - secure pipe daemon
-encrypted connections to the target socket.
+Take unencrypted connections from the
+.I source socket
+and send
```
why the linebreak?

TimWolla replied:
Forgot to remove the line break.

Assuming your review is finished (last message was half an hour ago): I just fixed the issues.

gperciva commented:
Looks good to me.
